server/uptime-calculator.js

```diff
@@ -606,7 +654,11 @@ class UptimeCalculator {
             avgPing = totalPing / total.up;
         }
 
-        uptimeData.uptime = total.up / (total.up + total.down);
+        if (total.up + total.down === 0) {
+            uptimeData.uptime = 0;
```

Comment:
I think setting it to `undefined`/`null` and handling this in the frontend might be a better approach, but this is a good enough interims solution ^^